src/styles/components/card.css

```diff
@@ -0,0 +1,95 @@
+.card {
+  width: 200px;
+
+  padding: 32px 16px 16px;
+
+  background-color: var(--color-white);
+  border: 1px solid var(--color-concrete);
+  border-radius: 5px;
+}
+
+.card__image {
+  display: block;
+
+  width: 160px;
+  height: 134px;
+
+  margin: 0 auto 40px;
+
+  object-fit: contain;
+}
+
+.card__heading {
+  margin: 0 0 4px;
```

Comment:
Ok, but I think it would be better for readability to reset default margins somewhere on the top of file (or even in separate file) and set only bottom margin here

Reply:
To be honest also think about it, but i dont really like to use tags. But if i create class for reset, he is gonna be block (BEM), but block can't have margins, etc.. So i'm just can't decide which is more correct. Next time will use tag for reset like other.